fix(task): required lookup and list field for task completion

SamuelBlais opened the pull request:
now correctly show when completing a task if not already filled.
 Also always hide description field and always show comment fields.

FD-8910 FD-9495 CU-fgqvrz

MichaelTurcotte committed:
fix(task): required lookup and list field for task completion
 now correctly show when completing a task if not already filled.
 Also always hide description field and always show comment fields.

FD-8910 FD-9495 CU-fgqvrz

vdussault reviewed (commented):
Plutôt que d'initialiser descriptionField à null, je changerais la valeur retournée par buildDescriptionField.

Voir avec Gabriel pourquoi il avait ajouté la description (révision 483ffae28063377ae3981d01267e5021e88e2f98 dans l'ancien repo).

Voir avec Gabriel, Nicolas et Rabab s'ils se souviennent pourquoi buildCommentField retournait une valeur nulle (révision 54a24d8e28ea471236cf08b62e391432a0bf9977).

SamuelBlais committed:
fix(task): required lookup and list field for task completion

now correctly show when completing a task if not already filled.
Also make description field read-only if shown and always show comment fields.

FD-8910 FD-9495 CU-fgqvrz
